lib/srte.h

```diff
@@ -19,6 +19,23 @@ enum zebra_sr_policy_status {
 	ZEBRA_SR_POLICY_DOWN,
 };
 
+/* SR types. */
+enum sr_types_t {
```

Comment:
Typically XXX_t refers to a typedef.  This is not a typedef.   This should be `enum sr_types` or `enum sr_types_enum`

Reply:
fixed to enum sr_types